plugins/modules/foreman_compute_resource.py

```diff
@@ -185,19 +171,15 @@ def main():
             locations=dict(type='list'),
             organizations=dict(type='list'),
             state=dict(type='str', default='present', choices=['present', 'absent', 'present_with_defaults']),
-
-            # Deprecated provider's specific params, use nested keys in provider_params param instead
             provider_auth=dict(type='dict'),
```

Comment:
any reason not to remove this too? as it's deprecated?

@bagasse what do you think? IIRC that's your code initially

Reply:
I moved this to provider_params some time ago because some providers needs some specific params other than auth (datacenter, quota...) when I improved oVirt support. I don't know the deprecation policy for foreman ansible modules, but since the commit that introduced it is more than 5 month old, I think we can remove this param.

Reply:
Yeah, we don't have a policy (yet). We should once we start shipping this in a versioned manner. But until then, I too think just remove it is the right path.

Reply:
You should also update module documentation documentation accordingly.